src/cloud-cli/meltano/cloud/README.md

```diff
@@ -0,0 +1,85 @@
+# Meltano Cloud CLI
+
+`meltano-cloud` is a standalone CLI tool for interacting with the Meltano Cloud API.
+
+## Configuration
+
+`meltano-cloud` can be configured via
+`meltano-cloud config set <setting name> <setting value>`.
+
+### Configuration File
+
+Meltano Cloud configuration settings and credentials are stored in a JSON file.
+`meltano-cloud` searches for this configuration file using [platformdirs](https://github.com/platformdirs/platformdirs).
+
+If a configuration file is found but the configuration it contains
+is invalid, `meltano-cloud` will raise an error rather than continuing its search.
+
+### Settings
+
+The following settings are configurable:
+
+#### `auth_callback_port`
+
+As part of authentication, `meltano-cloud` runs a lightweight, short-lived HTTP
+server on `localhost` to handle authentication callbacks. By default, this
+server runs on port 9999 but can be configured to another port using this setting.
+
+- Command line flag: `--auth-callback-port`
+- Environment variable: `MELTANO_CLOUD_AUTH_CALLBACK_PORT`
+
+#### `base_url`
+
+This is the base URL used for API calls to Meltano Cloud, `https://api.meltano.cloud`
+by default.
+
+- Command line flag: `--base-url`
+- Environment variable: `MELTANO_CLOUD_BASE_URL`
+
+#### `base_auth_url`
+
+This is the base URL used for authentication calls to Meltano Cloud,
+`https://auth.meltano.cloud` by default.
+
+- Command line flag: `--base-auth-url`
+- Environment variable: `MELTANO_CLOUD_BASE_AUTH_URL`
+
+#### `organization_id`
+
+This is the default ID for the Meltano Cloud organization to use in interacting with
+the Meltano Cloud API.
+
+- Command line flag: `--organization-id`
+- Environment variable: `MELTANO_CLOUD_ORGANIZATION_ID`
```

Comment:
Do we need this to be configurable? 

If not, I prefer to drop it and avoid needing to support permanent user-facing organization IDs, which aren't yet a thing we're supporting/declaring for Cloud users here: https://github.com/meltano/cloud-docs/blob/main/docs/concepts.md

Reply:
I'd like to leave it configurable for ease of testing, but a middle path might be to remove from docs and not explicitly support it.

Reply:
That makes sense. Wdyt of renaming with an "internal" prefix like "MELTANO_CLOUD_INTERNAL_ORG_ID"?

In that case, we can worry less about whether it's in the docs or not, since it's more self-documenting as non-user-facing.